Sources/TuistGraph/Models/ResourceSynthesizerPlugin.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import TSCBasic
+
+/// Resource synthesizer plugin model
+public struct ResourceSynthesizerPlugin: Equatable {
+    /// Name of the plugin
+    public let name: String
```

Comment:
true, this is unnecessary duplication, I'll remove this field

Reply:
actually - false 😅 `ResourceSynthesizerPlugin` is a part of `Plugins` that contains *all* plugins and thus does not contain its name. This might be worth to refactor imho, so this duplication is needed but is a little out of scope for this PR